### ISSUE TITLE
Tab indention isn't caught for lines beginning with . when indent is set to use spaces

### ISSUE THREAD

**Issue by treythomas123** [eslint/eslint#5540]
**What version of ESLint are you using?**
2.3.0

**What did you do? Please include the actual source code causing the issue.**

<img width="264" alt="screen shot 2016-03-10 at 1 40 17 pm" src="https://cloud.githubusercontent.com/assets/2589390/13683750/b0146218-e6c5-11e5-8fd6-06a6c413f0be.png">

**What did you expect to happen?**

Line 6 should cause an error because it has tab indention, and I specified `indent: [2, 4]`, which should enforce spaces. Note that this also isn't caught by the rule `no-mixed-spaces-and-tabs`, because that rule only checks for tabs and spaces mixed on the same line. Line 6 has only tabs.

**What actually happened? Please include the actual, raw output from ESLint.**

Eslint passes with no errors.

**Comment by nzakas** [comment]
That's strange. @byk @gyandeeps thoughts?

**Comment by BYK** [comment]
Verified on master with the following snippet:

```js
{
            code:
            "function test() {\n" +
            "    Promise\n" +
            "        .resolve()\n" +
            "\t\t.then( doSomething )\n" +
            "        .then( doSomethingElse );\n" +
            "}",
            parserOptions: { ecmaVersion: 6 },
            options: [4]
        },
```

**Comment by gyandeeps** [comment]
This is not a bug because this falls under the multi-line case. Our indent rule doesnt check indentation for multi-line stuff.
related: https://github.com/eslint/eslint/issues/5073 n others

**Comment by treythomas123** [comment]
Could the `indent` rule be updated to catch when spaces are expected but tabs are found (or vice versa), without enforcing a certain indention level for the multi-line statement? That seems like it'd be easier to implement than full support for the different possible multi-line chaining styles.

If you're open to that, I can look into opening a PR.

**Comment by BYK** [comment]
@treythomas123 interesting thought but I'd prefer not complicating the indent rule s it is. I am working on a rewrite. Can't give estimates yet but I'm not too far from figuring it out. You may check my trials on the `new-indent` branch.

**Comment by gyandeeps** [comment]
@treythomas123 meanwhile u can use `no-tabs-in-file` rule fro this plugin: https://github.com/gyandeeps/eslint-plugin-ideal

**Comment by ilyavolodin** [comment]
@gyandeeps If I'm correct we are now checking indentation for multiline statements, however, this issue is still not being reported. I just tested it in online demo and it lints without errors. Should this be marked as accepted?

**Comment by gyandeeps** [comment]
Not yet @ilyavolodin .

**Comment by kaicataldo** [comment]
@ilyavolodin @gyandeeps This has been open a long time without being accepted. Can we go ahead and close it?

**Comment by nzakas** [comment]
I think @ilyavolodin was going to combine open indent issues into one. Did that happen?

**Comment by ilyavolodin** [comment]
No, sorry, I still haven't gotten around to doing that:-( It's on my to-do list.

**Comment by nzakas** [comment]
Okay, let's leave this open until that happens (btw, I'm sure if anyone wants to take that off @ilyavolodin's plate, he'd appreciate it).

**Comment by ilyavolodin** [comment]
Absolutely! If somebody has time to go through indent issue, please don't hesitate to do that!

**Comment by kaicataldo** [comment]
@nzakas @ilyavolodin done! https://github.com/eslint/eslint/issues/7274

**Comment by not-an-aardvark** [comment]
I think this issue has been fixed, so I'm going to close it. You can now use the `MemberExpression` option to check the indentation of MemberExpression nodes, and the indent checker will report a useful error if it finds tabs when it expected spaces.
